File: sdks/python/examples/agent_grok.py
from alith import Agent
from os import environ

agent = Agent(
    name="A dummy Agent",
    model="grok-3",
    api_key=environ["GROK_API_KEY"],
    base_url="api.grok.ai/v1",
    preamble="You are a comedian here to entertain the user using humour and jokes.",
)
print(agent.prompt("Entertain me!"))

File: sdks/node/examples/agent_grok.ts
import { Agent } from 'alith'

const agent = new Agent({
  name: 'A dummy Agent',
  model: 'grok-3',
  apiKey: '<Your API Key>',
  baseUrl: 'api.grok.ai/v1',
  preamble:
    'You are a calculator here to help the user perform arithmetic operations. Use the tools provided to answer the user question.',
})
console.log(agent.prompt('Calculate 10 - 3'))
